addons/im_livechat/static/src/embed/common/store_service_patch.js

```diff
@@ -19,19 +18,10 @@ patch(Store.prototype, {
             }
             return;
         }
-        const messagingData = {
-            "discuss.channel": [],
-            Store: { settings: {} },
-        };
-        if (session.livechatData?.options.current_partner_id) {
-            messagingData.Store.current_partner = {
-                id: session.livechatData.options.current_partner_id,
-            };
+        if (livechatService.savedState?.store) {
+            const { Thread = [] } = this.store.insert(livechatService.savedState.store);
+            livechatService.thread = Thread[0];
         }
```

Comment:
Note: `current_partner` and `current_partner_id` is dead code, no longer used since livechat refactoring.